test/unittests/evm_eof_test.cpp

```diff
@@ -203,17 +208,365 @@ TEST_P(evm, eof1_codecopy_out_of_bounds)
 {
     // 4 bytes out of container bounds - result is implicitly 0-padded
     rev = EVMC_CANCUN;
-    auto code = eof1_bytecode(bytecode{23} + 0 + 0 + OP_CODECOPY + ret(0, 23));
+    auto code = eof1_bytecode(bytecode{35} + 0 + 0 + OP_CODECOPY + ret(0, 35), 3);
+
+    execute(code);
+    EXPECT_STATUS(EVMC_SUCCESS);
+    EXPECT_EQ(bytes_view(result.output_data, result.output_size),
+        "ef0001010004020001000c03000000000000036023600060003960236000f300000000"_hex);
+
+    code = eof1_bytecode(bytecode{39} + 0 + 0 + OP_CODECOPY + ret(0, 39), 3, "deadbeef");
+
+    execute(code);
+    EXPECT_STATUS(EVMC_SUCCESS);
+    EXPECT_EQ(bytes_view(result.output_data, result.output_size),
+        "ef0001010004020001000c03000400000000036027600060003960276000f3deadbeef00000000"_hex);
+}
+
+TEST_P(evm, eof2_rjump)
```

Comment:
`eof2_*` tests should be renamed

Reply:
What do you mean?

Reply:
RJUMP is certainly not part of EOF2, and overall test names are inconsistent - there're prefixes `eof1_`, `eof2_`, `eof_` and some with no prefix. I would vote for leaving only `eof_`

Reply:
I left it with `eof1_` prefix as it was already dominating.